tests/test_converter.py

```diff
@@ -1095,87 +1156,6 @@ def circuit_native_pennylane(angle):
             rotation_angle
         )
 
-    def test_passing_parameters_new_interface_args(self, qubit_device_2_wires):
```

Comment:
these are just copy-pasted further down to the section for testing passing parameters